src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -242,6 +246,21 @@ function dependencyHasCycle(currentBuildData: BuildData, buildData:BuildData[],
   return [false, undefined];
 }
 
+function calculateNodeTimeToInclusion(build:Build, buildData:BuildData[], inclusionTimeOfParent:number): void {
+  if (build.dependencies)
+  {
```

Comment:
This is the weird semi-colon thing we were talking about - if the open braces are on new lines the compiler can do stupid things in some cases.